Ginger/GingerCoreNET/RunLib/CLILib/CLIHelper.cs

```diff
@@ -315,6 +321,34 @@ private bool CheckforReRunConfig()
                             Result = false;
                         }
                     }
+                    else if(mRunSetConfig.ReRunConfigurations.RerunLevel == eReRunLevel.Runner)
+                    {
+                        List<AccountReportRunner> accountReportRunnerList = accountReportApiHandler.GetRunnerExecutionDataFromCentralDB((Guid)mRunSetConfig.ReRunConfigurations.ReferenceExecutionID);
+                        if (accountReportRunnerList != null)
+                        {
+                            if(accountReportRunnerList.Any(x=> x.RunStatus.Equals(eRunStatus.Failed.ToString(),StringComparison.CurrentCultureIgnoreCase)))
+                            {
+                                var FailedRunnerGuidList = accountReportRunnerList.Where(x => x.RunStatus.Equals(eRunStatus.Failed.ToString(), StringComparison.CurrentCultureIgnoreCase)).Select(x => x.Id).ToList();
+                                foreach (GingerRunner runner in mRunsetExecutor.RunSetConfig.GingerRunners)
+                                {
+                                    if (!FailedRunnerGuidList.Contains(runner.Guid))
```

Comment:
Do we need to maintain the list of FaileRunnerGuifList?

Reply:
yes we are getting runner list from Account report to identify which runner need to Rerun and other need to set as Deactive